src/components/menu/menu-full-screen/menu-full-screen.spec.js

```diff
@@ -65,65 +66,69 @@ describe("MenuFullscreen", () => {
   });
 
   describe("styling", () => {
-    it("matches the expected as default", () => {
-      assertStyleMatch(
-        {
-          position: "fixed",
-          top: "0",
-          bottom: "0",
-          backgroundColor: baseTheme.menu.light.background,
-          zIndex: `${baseTheme.zIndex.fullScreenModal}`,
-          visibility: "hidden",
-          left: "-100vw",
-          transition: "all 0.3s ease",
-        },
-        wrapper
-      );
-
-      ["a", "button", "div"].forEach((el) => {
+    it.each(["light", "white", "dark", "black"])(
+      "matches the expected as default",
+      (menuType) => {
+        wrapper = render({ menuType });
         assertStyleMatch(
           {
-            fontSize: "16px",
+            position: "fixed",
+            top: "0",
+            bottom: "0",
+            backgroundColor: menuConfigVariants[menuType].background,
+            zIndex: `${baseTheme.zIndex.fullScreenModal}`,
+            visibility: "hidden",
+            left: "-100vw",
+            transition: "all 0.3s ease",
           },
-          wrapper,
-          { modifier: el }
+          wrapper
         );
-      });
 
-      assertStyleMatch(
-        {
-          position: "absolute",
-          zIndex: "1",
-          right: "16px",
-          top: "8px",
-        },
-        wrapper.find(StyledMenuFullscreenHeader),
-        { modifier: `${StyledIconButton}` }
-      );
+        ["a", "button", "div"].forEach((el) => {
+          assertStyleMatch(
+            {
+              fontSize: "16px",
+            },
+            wrapper,
+            { modifier: el }
+          );
+        });
 
-      assertStyleMatch(
-        {
-          paddingTop: "16px",
-          paddingBottom: "16px",
-        },
-        wrapper.find(StyledMenuItem)
-      );
-    });
+        assertStyleMatch(
+          {
+            position: "absolute",
+            zIndex: "1",
+            right: "16px",
+            top: "8px",
+          },
+          wrapper.find(StyledMenuFullscreenHeader),
+          { modifier: `${StyledIconButton}` }
+        );
+
+        assertStyleMatch(
+          {
+            paddingTop: "16px",
+            paddingBottom: "16px",
+          },
+          wrapper.find(StyledMenuItem)
+        );
+      }
+    );
 
     it.each(["light", "white", "dark", "black"])(
       "applies the expected styling when `menuType` is %s",
       (menuType) => {
         wrapper = render({ menuType });
         assertStyleMatch(
           {
-            backgroundColor: baseTheme.menu[menuType].background,
+            backgroundColor: menuConfigVariants[menuType].background,
           },
           wrapper
         );
 
         assertStyleMatch(
```

Comment:
It may be helpful to reorganise the assertions in this file into separate tests under a `describe.each()` for readability